src/app/core/store/collections/collection.ts

```diff
@@ -27,13 +27,21 @@ export interface Attribute {
   name: string;
 
   constraint?: Constraint;
+  function?: AttributeFunction;
 
   usageCount?: number;
   intermediate?: boolean;
 
   correlationId?: string;
 }
 
+export interface AttributeFunction {
+  js?: string;
+  xml?: string;
+  errorReport?: string;
+  timestamp?: number;
```

Comment:
Is this a date and do we want to work with it on frontend? If so, it would make sense to have `Date` type here and convert the value.

Reply:
It is rather a technical value for read only on frontend and debugging purposes. Similar as in Blockly rules. There is no point in making it more complicated. We need to see that the value increased over time...